R/fix.R

```diff
@@ -49,14 +49,13 @@
 #' fix(destfile)
 #' cat(paste(readLines(destfile), collapse = "\n"))
 fix <- function(
-  path = ".",
-  linters = NULL,
-  exclude_path = NULL,
-  exclude_linters = NULL,
-  force = FALSE,
-  verbose = TRUE,
-  rerun = TRUE
-) {
+    path = ".",
+    linters = NULL,
+    exclude_path = NULL,
+    exclude_linters = NULL,
+    force = FALSE,
+    verbose = TRUE,
+    rerun = TRUE) {
```

Comment:
Can you remove the changes related to formatting? This project uses `air` for automatic formatting (I should probably have this written somewhere). You can install it as described [here](https://posit-dev.github.io/air/cli.html) and run `air format .` from the command line in the folder.